Plugins/Creature/Creature.cpp

```diff
@@ -2147,12 +2157,342 @@ ArgumentStack Creature::JumpToLimbo(ArgumentStack&& args)
 {
     if (auto *pCreature = creature(args))
     {
-        if (!pCreature->m_bPlayerCharacter && !pCreature->m_pStats->m_bIsPC && !pCreature->m_pStats->m_bIsDM) {
+        if (!pCreature->m_bPlayerCharacter && !pCreature->m_pStats->m_bIsPC && !pCreature->m_pStats->m_bIsDM)
+        {
             pCreature->RemoveFromArea();
             Utils::GetModule()->AddObjectToLimbo(pCreature->m_idSelf);
         }
     }
     return Services::Events::Arguments();
 }
 
+void Creature::InitCriticalMultiplierHook()
+{
+    static NWNXLib::Hooking::FunctionHook* pGetCriticalHitMultiplier_hook;
+    if (!pGetCriticalHitMultiplier_hook)
+    {
+        g_plugin->GetServices()->m_hooks->RequestExclusiveHook<Functions::_ZN17CNWSCreatureStats24GetCriticalHitMultiplierEi>(
+            +[](CNWSCreatureStats *pThis, bool bOffHand = false) -> int32_t
```

Comment:
The `BOOL` type in the API is actually an int32_t, so `bool bOffHand = false` should be `int32_t bOffHand = false`.